.github/workflows/e2e-starter.yml

```diff
@@ -0,0 +1,71 @@
+# SPDX-License-Identifier: Apache-2.0
+
+name: E2E Starter
+
+on:
+  # run against every merge commit to 'main' and release branches
+  push:
+    branches:
+      - main
+      - release-*
+  # only run on PRs that touch certain regex paths
+  pull_request:
+    branches:
+      - main
+      - release-*
+    paths:
+      # note this should match the merging criteria in 'mergify.yml'
+      - '**.py'
+      - 'pyproject.toml'
+      - 'requirements**.txt'
+      - 'scripts/basic-workflow-tests.sh' # Used by this workflow
+      - 'scripts/test-data/**'
+      - '.github/workflows/e2e-nvidia-t4-x1-gate.yml' # This workflow
+      - '.github/workflows/e2e-nvidia-t4-x1.yml' # Follow-on workflow
+
+env:
+  LC_ALL: en_US.UTF-8
+
+defaults:
+  run:
+    shell: bash
+
+permissions:
+  contents: read
+
+jobs:
+  e2e-status-checks:
+    if: ${{ github.repository == 'instructlab/instructlab' }}
+    permissions:
+      checks: read
+    uses: ./.github/workflows/status-checks.yml
+    with:
+      job_ids: >- # Space-separated job ids to wait on for status checks
+        DCO
+        actionlint
+        markdown-lint
+        shellcheck
+        spellcheck
+        lint-workflow-complete
```

Comment:
I may be misunderstanding the implementation, but seems the flow here is:

`Lints pass` => `Unit/Functional/E2E tests run`

If I'm correct, do we want that? Or something like

`Lints pass` => `Unit/Functional tests pass` => `E2E starts`

Reply:
To be clear, I realize the original issue calls for the former implementation - not opposed to that route, just figured I'd bring up the topic

Reply:
At this stage I think the current ordering is fine. We can refine to run e2e after test in a followup PR if desired.

Reply:
SGT, :+1: